src/audio/pipeline/pipeline-graph.c

```diff
@@ -344,6 +344,8 @@ int pipeline_reset(struct pipeline *p, struct comp_dev *host)
 	if (ret < 0) {
 		pipe_err(p, "pipeline_reset(): ret = %d, host->comp.id = %u",
 			 ret, dev_comp_id(host));
+	} else {
+		p->status = COMP_STATE_READY;
```

Comment:
Can we have a comment here like the commit message.

Reply:
sure